jans-cedarling/cedarling/src/authz/mod.rs

```diff
@@ -278,7 +279,7 @@ impl Authz {
             resource: resource_uid.to_string(),
             decision: result.decision.into(),
             tokens: tokens_logging_info,
-            decision_time_ms: elapsed_ms,
+            elapsed_micro_sec,
```

Comment:
changing `decision_time_ms` to `elapsed_micro_sec` gets rid of information on what is being measured. why not just change the unit?

```suggestion
            decision_time_usec,
```

Reply:
You are right, changed https://github.com/JanssenProject/jans/pull/10920/commits/84c565bae552ab6525723f106faf050b42fb5426